### DISCUSSION TITLE
Data leaking in the csv file

### DISCUSSION THREAD

**Discussion by HaDoyle12** [gavinr/github-csv-tools#103] [Help]
When pulling from a private repo, I am getting info from the issues in the rows of the csv file, this is causing issues cause we have to clean up the file before using. I also am not sure if all of the issues are being downloaded. This is a newer issue when running the tool. I am not even sure what could be causing the problem.

**Comment by gavinr** [comment]
resolution: https://github.com/gavinr/github-csv-tools/issues/104
